packages/terra-date-picker/tests/nightwatch/components/DatePickerDefaultDateExcluded.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Base from 'terra-base';
+import DatePicker from '../../../lib/DatePicker';
+
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
+
+const DatePickerDefaultDateExcluded = () => (
+  <Base locale={locale}>
```

Comment:
In some examples you use the IntlProvider and in these you use base. Can it be the same for all tests?

Reply:
This is following the same way as the other components.